android/src/main/kotlin/com/example/flutterimagecompress/exif/ExifKeeper.java

```diff
@@ -87,17 +89,34 @@ public ByteArrayOutputStream writeToOutputStream(Context context, ByteArrayOutpu
       fileOutputStream.close();
 
       ByteArrayOutputStream newStream = new ByteArrayOutputStream();
```

Comment:
I think that writing this will add a stream object, which may bring new problems in certain scenarios, but this can solve the memory leak, I think it is feasible.